### DISCUSSION TITLE
Query regarding the capabilities of the project

### DISCUSSION THREAD

**Discussion by sripor62** [pouriyajamshidi/flat#33] [Q&A]
Can this tool be used to measure latency of a packet between two interfaces?

**Comment by pouriyajamshidi** [comment]
@sripor62 I believe so. Please give it a shot and revert.

**Comment by sripor62** [comment]
I am facing some issues .Is there a way to contact you directly through mail or something else.If you could help

**Comment by pouriyajamshidi** [comment]
You could try LinkedIn or Discord

pouriya.jamshidi

**Comment by pouriyajamshidi** [comment]
Ultimately, you can run multiple instances of `flat` at the same time for different NICs without any issues.
